resources/translations/fi.edn

```diff
@@ -216,36 +216,42 @@
+                                 :subject-to-handler "Hakemus hylätty (%3)"
+                                 :message-to-handler "Hyvä %1,\n\n%2 on hylännyt käyttäjän %4 hakemuksen %3.\n\nVoit tarkastella hakemusta ja päätöstä: %6"}
+          :application-closed {:subject-to-applicant "Hakemuksesi on suljettu (%3)"
                                :message-to-applicant "Hyvä %1,\n\nHakemuksesi %3 on suljettu.\n\nTarkastele hakemusta: %5"
+                               :subject-to-handler "Hakemus suljettu (%3)"
                                :message-to-handler "Hyvä %1,\n\n%2 on sulkenut käyttäjän %4 hakemuksen %3.\n\nTarkastele hakemusta: %5"}
           :application-returned {:subject "Hakemus palautettu (%3)"
-                                 :message-to-applicant "Hyvä %1,\n\nHakemuksesi %3 on palautettu.\n\nTarkastele hakemusta: %5"
-                                 :message-to-handler "Hyvä %1,\n\n%2 on palauttanut käyttäjän %4 hakemuksen %3.\n\nTarkastele hakemusta: %5"}
+                                 :message-to-applicant "Hyvä %1,\n\nHakemuksesi %3 on palautettu muokattavaksi.\n\nVoit muokata hakemusta ja lähettää sen uudelleen: %6"
+                                 :message-to-handler "Hyvä %1,\n\n%2 on palauttanut käyttäjän %4 hakemuksen %3.\n\nTarkastele hakemusta: %6"}
           :application-licenses-added {:subject "Hakemukseen lisätty käyttöehtoja (%3)"
-                                       :message-to-applicant "Hyvä %1,\n\nHakemukseesi %3 on lisätty käyttöehtoja. Käyttöehdot täytyy hyväksyä ennen käyttöoikeuden saamista.\n\nTarkastele hakemusta: %5"
-                                       :message-to-handler "Hyvä %1,\n\n%2 on lisännyt käyttöehtoja käyttäjän %4 hakemukseen %3.\n\nTarkastele hakemusta: %5"}
-          :application-submitted {:subject "Hakemus lähetetty (%4: %3)"
-                                  :message "Hyvä %1,\n\n%4 on lähettänyt käyttöoikeushakemuksen (%3): %5.\n\nTarkastele hakemusta: %6"}
-          :comment-requested {:subject "Katselmointipyyntö (%2: %3)"
-                              :message "Hyvä %1,\n\n%2 on pyytänyt katselmoimaan hakemuksen %3.\n\nKommentoi: %6"}
-          :commented {:subject "Hakemus katselmoitu (%2: %3)"
-                      :message "Hyvä %1,\n\n%2 on katselmoinut hakemuksen %3.\n\nTarkastele hakemusta: %6"}
-          :decided {:subject "Päätös tehty (%2: %3)"
-                    :message "Hyvä %1,\n\n%2 on tehnyt päätöksen hakemuksesta %3.\n\nTarkastele hakemusta: %6"}
-          :decision-requested {:subject "Päätöspyyntö (%2: %3)"
-                               :message "Hyvä %1,\n\n%2 on pyytänyt sinua tekemään päätöksen hakemuksesta %3.\n\nTarkastele hakemusta: %6"}
+                                       :message "Hyvä %1,\n\n%2 pyytää hyväksyntääsi uusiin käyttöehtoihin hakemukselle %3.\n\nVoit tarkastella hakemusta ja hyväksyä käyttöehdot: %6"}
```

Comment:
I think "pyytää hyväksyntääsi uusille käyttöehdoille hakemukseen" would be slightly better.

Reply:
done